content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md

```diff
@@ -0,0 +1,303 @@
+---
+aliases:
+- /head/generating-a-node-cordform.html
+- /HEAD/generating-a-node-cordform.html
+date: '2020-04-07T12:00:00Z'
+menu:
+  corda-os-4-8:
+    identifier: corda-os-4-8-cordform-plugin
+    parent: corda-os-4-8-generating-a-node
+    weight: 1000
+tags:
+- cordform
+- node
+title: Cordform plug-in
+---
+
+# Cordform plug-in
+
+Corda provides two `gradle` plug-ins called `Cordform` and `Dockerform`. They both allow you to run tasks that automatically generate and configure a local set of nodes for testing and demonstration purposes. This page contains information about the operation of the Cordform plug-in. Visit the [Dockerform](../../../../../en/platform/corda/4.8/open-source/generating-a-node-dockerform.md) page for Dockerform configuration options.
+
+* A `Cordform` task creates nodes in the `build/nodes` directory. The example `Cordform` task used in this document creates three nodes: `Notary`, `PartyA`, and `PartyB`, however you are free to spin up more nodes, specify what nodes you need on the network, change node names, and update node configurations.
+* `Cordform` tasks require you to deploy each Corda node and database separately.
+
+## Tasks using the Cordform plug-in
+
+Run this example task to create the following three nodes in the `build/nodes` directory:
+
+A `Notary` node, which:
+
+* Provides a validating Notary service.
+* Runs the `corda-finance` CorDapp.
+
+`PartyA` and `PartyB` nodes, each of which:
+
+* Does not provide any services.
+* Runs the `corda-finance` CorDapp.
+* Has an RPC (Remote Procedure Call) user (`user1`), which enables you to log in the node via RPC.
+
```

Comment:
Your bulleted lists might look a bit strange here. Party A / notary / etc should be bulleted and then the rest indented.